website/common/locales/en/groups.json

```diff
@@ -111,6 +111,20 @@
   "sortHabitrpgLastLoggedIn": "Sort by last time user logged in",
   "ascendingSort": "Sort Ascending",
   "descendingSort": "Sort Descending",
+
+  "sortClass": "Class",
+  "sortBackground": "Background",
+  "sortDateJoinedAsc": "Earliest Date Joined",
+  "sortDateJoinedDesc": "Latest Date Joined",
+  "sortLoginAsc": "Earliest Login",
+  "sortLoginDesc": "Latest Login",
+  "sortLevelAsc": "Lowest Level",
+  "sortLevelDesc": "Highest Level",
+  "sortNameAsc": "Name (A - Z)",
+  "sortNameDesc": "Name (Z - A)",
+  "sortTierAsc": "Lowest Tier",
+  "sortTierDesc": "Highest Tier",
```

Comment:
Will these no longer be needed?

https://github.com/kartikadur/habitica/blob/95a7452c8cd3d75d3c39a1a3ffe3b2e7bb244ef6/website/common/locales/en/groups.json#L105-L113

Reply:
Hmm yeah, it looks like there are some duplicates here

Reply:
@paglias thanks for letting me know. I wasn't sure if they were used anywhere else.